sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -17,9 +17,7 @@
 
 package org.apache.spark.sql.execution.command
 
-import java.io.File
```

Comment:
This looks unrelated?

Reply:
just unused import, I think no need raise a new PR for unused import.